ts/screens/onboarding/OnboardingTosScreen.tsx

```diff
@@ -120,64 +108,65 @@ const OnboardingTosScreen = () => {
           text: I18n.t("global.buttons.exit"),
           style: "default",
           onPress: () => {
-            trackTosUserExit(getFlowType(true, isFirstOnBoarding));
+            trackTosUserExit(flow);
             dispatch(abortOnboarding());
           }
         }
       ]
     );
 
-  const store = useStore();
+  useHeaderSecondLevel({
+    title: "",
+    supportRequest: true,
+    goBack: handleGoBack,
+    contextualHelpMarkdown,
+    faqCategories: ["privacy"]
+  });
 
   return (
     <LoadingSpinnerOverlay isLoading={isLoading || isUpdatingProfile}>
-      <BaseScreenComponent
-        goBack={handleGoBack}
-        contextualHelpMarkdown={contextualHelpMarkdown}
-        faqCategories={["privacy"]}
-        headerTitle={I18n.t("onboarding.tos.headerTitle")}
-      >
-        <SafeAreaView style={styles.webViewContainer}>
-          <View style={styles.titlePadding}>
-            <H1
-              accessible={true}
-              accessibilityRole="header"
-              weight="Bold"
-              testID={"screen-content-header-title"}
-              color={"bluegreyDark"}
-            >
-              {I18n.t("profile.main.privacy.privacyPolicy.title")}
-            </H1>
-          </View>
-          {!hasAcceptedCurrentTos && (
-            <View
-              style={[styles.alert, styles.titlePadding]}
-              testID={"currentToSNotAcceptedView"}
-            >
-              <Body testID={"currentToSNotAcceptedText"}>
-                {hasAcceptedOldTosVersion
+      <SafeAreaView edges={["bottom"]} style={IOStyles.flex}>
+        <View style={IOStyles.horizontalContentPadding}>
+          <H1
```

Comment:
Isn't this an [`H2`](https://www.figma.com/file/BEXvjQmBkMRRFiNcMgE8g3/Login-Veloce?type=design&node-id=5093-31126&mode=design&t=B9xAKMobJih2k6ZB-4)?